### PR TITLE
optimize #69 display exception detail & enable NebulaDaoBasic to support space switching

### PR DESCRIPTION
**Pull request by CorvusYe**

### CODE REVIEW

**Review by wey-gu — APPROVED** [review]

File: src/main/java/org/nebula/contrib/ngbatis/proxy/MapperProxy.java [hunk 1]
```diff
@@ -218,16 +218,18 @@ public static ResultSet executeWithParameter(ClassModel cm, MethodModel mm, Stri
       }
 
       localSession = ENV.getDispatcher().poll();
-      gql = qlWithSpace(localSession, gql, getSpace(cm, mm));
+      String currentSpace = getSpace(cm, mm);
+      gql = qlWithSpace(localSession, gql, currentSpace);
```
Comment (Nicole00) [inline src/main/java/org/nebula/contrib/ngbatis/proxy/MapperProxy.java:222]: What if there's `USE space` in gql? is there any check for gql

Comment (CorvusYe) [reply src/main/java/org/nebula/contrib/ngbatis/proxy/MapperProxy.java:222]: `org.nebula.contrib.ngbatis.proxy.MapperProxy#findLastSpaceFromGql`
This method will find the space which is used lastest, and set to current local session's space.

Comment (CorvusYe) [reply src/main/java/org/nebula/contrib/ngbatis/proxy/MapperProxy.java:222]: @Nicole00 
Whether this usage is appropriate from the perspective of upstream project ?

Comment (CorvusYe) [reply src/main/java/org/nebula/contrib/ngbatis/proxy/MapperProxy.java:222]: > What if there's `USE space` in gql? is there any check for gql

Developers can use `USE space` in ngql, and ngbatis does not check.
I seem to have misunderstood just now. Sorry

Comment (Nicole00) [reply src/main/java/org/nebula/contrib/ngbatis/proxy/MapperProxy.java:222]: > `org.nebula.contrib.ngbatis.proxy.MapperProxy#findLastSpaceFromGql` This method will find the space which is used lastest, and set to current local session's space.

Yeah, I see the logic of MapperProxy. And I have confuse why we maintain the current space of session in LocalSession.

Comment (CorvusYe) [reply src/main/java/org/nebula/contrib/ngbatis/proxy/MapperProxy.java:222]: According to my usage, the Nebula session itself has a spatial state. Here, to keep the state of the locally scheduled session consistent with that of the Nebula session.

It is possible to switch the space according to the interface each time, but I am not sure whether the cost is high, so the local session also maintains the same space as the Nebula session.

**Pull request merged**